packages/fluentui/react-northstar/src/themes/teams/components/Attachment/attachmentStyles.ts

```diff
@@ -130,7 +131,6 @@ const attachmentStyles: ComponentSlotStylesPrepared<AttachmentProps, AttachmentV
     display: 'block',
     bottom: 0,
     left: 0,
-    width: `${p.progress}%`,
```

Comment:
I moved this to `style` prop on `progress` as in `Slider` component as obviously we don't want to have one hundled entries in cache to different values of `progress`